src/frameworks/CoreMedia/include/CoreMedia/CMTime.h

```diff
@@ -20,12 +20,20 @@
 #ifndef _CMTIME_H_
 #define _CMTIME_H_
 
+#include <CoreFoundation/CFBase.h>
 #include <stdint.h>
 
 typedef int64_t CMTimeValue;
 typedef int32_t CMTimeScale;
 typedef int64_t CMTimeEpoch;
-typedef uint32_t CMTimeFlags;
+typedef CF_OPTIONS(CFOptionFlags, CMTimeFlags) {
```

Comment:
Make sure the type matches what is specified [on the developer website](https://developer.apple.com/documentation/coremedia/cmtimeflags?language=objc).

```suggestion
typedef CF_OPTIONS(uint32_t, CMTimeFlags) {
```

Reply:
Done